Source/WebCore/rendering/TextBoxPainter.cpp

```diff
@@ -138,8 +142,12 @@ void TextBoxPainter::paint()
         m_renderer.page().addRelevantRepaintedObject(m_renderer, enclosingLayoutRect(m_paintRect));
     }
 
-    if (shouldRotate)
-        m_paintInfo.context().concatCTM(rotation(m_paintRect, RotationDirection::Counterclockwise));
+    if (glyphRotation) {
+        RotationDirection backRotation = *glyphRotation == RotationDirection::Clockwise
```

Comment:
```suggestion
        auto backRotation = *glyphRotation == RotationDirection::Clockwise
```

Reply:
It won't compile without the explicit type.

Reply:
Actually, it seems that was a different thing that wouldn't compile. Fixed.